torch/_inductor/config.py

```diff
@@ -620,6 +620,9 @@ def decide_compile_threads() -> int:
 # Specify a file where we print out the profiling results.
 # None means we do not dump results to a file.
 profile_bandwidth_output = os.environ.get("TORCHINDUCTOR_PROFILE_OUTPUT", None)
+# Swithch to do_bench_using_profiling to exclude the CPU overheads
+_profile_option = os.environ.get("TORCHINDUCTOR_PROFILE_USING_PROFILING", "")
+profile_bandwidth_using_profiling = _profile_option != ""
```

Comment:
recommend following same method other configs use, i.e.
```
profile_bandwidth_using_profiling = os.environ.get("TORCHINDUCTOR_PROFILE_USING_PROFILING") == "1"
```
especially since this will currently set `profile_badwidth_using_profiling=True` if `TORCHINDUCTOR_PROFILE_USING_PROFILING=0`

Reply:
nit `profile_bandwidth_using_profiling` confused me for a second, maybe `profile_bandwidth_with_do_bench_using_profiling`?